packages/Material-Core.package/MDButton.class/class/defaultBackgroundColorToggled.st

```diff
@@ -0,0 +1,4 @@
+default properties
+defaultBackgroundColorToggled
+
+	^ MDColorTheme currentTheme secondaryColorDarkest
```

Comment:
Ich finde secondaryColorDarkest als Benennung nicht besonders intuitiv. Besser wäre vielleicht darkestSecondaryColor.
Kann aber verstehen, wenn man zum Beispiel bei allen Farben, die was mit secondary zu tun haben, das auch vorne dran stehen haben möchte.

Reply:
Genau der zweite Punkt war auch das, was wir wollten. Die Idee war, dass man alles zur secondaryColor in der Autovervollständigung 'secondary' eintippt.